frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/outline/outline_block_component.dart

```diff
@@ -151,7 +158,19 @@ class _OutlineBlockWidgetState extends State<OutlineBlockWidget>
               crossAxisAlignment: CrossAxisAlignment.start,
               mainAxisSize: MainAxisSize.min,
               textDirection: textDirection,
-              children: children,
+              children: [
+                Text(
```

Comment:
@Jayaprakash-dev Do you think it would be better to add placeholder text?

Reply:
I thought it would be better to have a placeholder.